### PR TITLE
Test runner

### PR DESCRIPTION
**Pull request by tarasevich**

### CODE REVIEW

File: BinarySearch/src/BinarySearchTestTest.java [hunk 1]
```diff
@@ -34,20 +34,32 @@ public void findElementInEmptyArray()
...
+    }
+
+    @Test
+    public void findSecondOfTwo() {
+      assertEquals(1, BinarySearch.find(a(1, 2), 2));
+    }
+
+    private static int[] a(int... array) {
+      return array;
+    }
```
Comment (sergeyenin) [inline BinarySearch/src/BinarySearchTestTest.java:64]: What is the advantage of a such semiosis?

**Comment by sergeyenin** [comment]
Could, you, please, explain the purpose of test runner, since it is not transparent?

File: BinarySearch/src/BinarySearchTestTest.java (hunk 1 above)
Comment (tarasevich) [inline BinarySearch/src/BinarySearchTestTest.java:64]: `a(1, 2)` is shorter than `new int [] {1, 2}`

**Comment by tarasevich** [comment]
to run tests, e.g.

```
./tst BinarySearch/src
```

**Comment by sergeyenin** [comment]
I meant, is there any another advantage besides running test outside of IDE?
I though, I missed it, probably.

**Comment by tarasevich** [comment]
not sure - advantage comparing to what?

**Comment by sergeyenin** [comment]
comparing to running tests within IDE.

**Comment by tarasevich** [comment]
i don't use an IDE

**Comment by den-sergeyev** [comment]
@sergeyenin, what is advantage of using IDE in such kind of project (there is no huge class indexes and no need to use autocomplete)?
However I see that IDE preserves you from deep understanding of low java project organization.

**Comment by sergeyenin** [comment]
@den-sergeyev I just asked what is the reason for such file)
I am not arguing of advantages/disadvantages of using or not IDE.

**Comment by tarasevich** [comment]
My point here is instead of building some casual skills of using IDEs I
prefer further developing of core skills related to such technologies like
bash, vim, linux etc

**Pull request merged**